src/test/scala/cromwell/engine/db/slick/TestSlickDatabase.scala

```diff
@@ -30,15 +30,15 @@ class TestSlickDatabase(configPath: String) {
       log.debug("Opening test connection setup for " + configPath)
       Database.forConfig("", databaseConfig)
     } flatMap { database =>
-      database.run(SimpleDBIO(_.connection.isValid(1))) recover {
-        case ex =>
-          log.error("Unable to connect to database under config: " + configPath, ex)
-          false
-      } andThen {
+      database.run(SimpleDBIO(_.connection.isValid(1))) andThen {
         case _ =>
           log.debug("Closing test connection setup for " + configPath)
           database.close()
       }
+    } recover {
+      case ex =>
+        log.warn("Unable to connect to database under config: " + configPath, ex)
```

Comment:
This is the second half of a two part boolean:

```scala
canConnect = testRequired || testDatabase.isValidConnection.futureValue
```

For the hsqldb databases, `testRequired` is `true`, so this short circuit won't activate, and the tests will throw errors.

I updated the method comment. If we would ever like to support users running cromwell on other dbs, we can add those as similar optional tests tests for those database engines too.